reggie/configs/primary_locale_names/county/virginia.json

```diff
@@ -188,8 +188,8 @@
     "id": "james city county"
   },
   {
-    "name": "King & Queen County",
-    "id": "king & queen county"
+    "name": "King and Queen County",
+    "id": "king and queen county"
```

Comment:
@zeldela - The string is actually "king & queen county" in the database, so the "id" field needs to remain "king & queen county". You can change the "name" field to whatever you think is best though (that's just the pretty-print name we want to send out in emails).

Reply:
got it -- changed this. does it need to be "&" in the virginia.yaml, too?

Reply:
the "county_names" in the yaml is never actually used in the code, so *shrug* it doesn't matter to me - But if you are using it to refer to database values then yes it should be "&"